app/src/main/java/de/danoeh/antennapod/fragment/FeedSettingsFragment.java

```diff
@@ -159,6 +162,26 @@ public void onDestroy() {
             }
         }
 
+        private void setupFeedAutoSkipPreference() {
+            findPreference("skipping").setOnPreferenceClickListener(preference -> {
```

Comment:
Sorry, I just noticed this line (otherwise, I would have written that earlier). Please use a constant here, just like the other preferences in this class.

Reply:
No worries will change